core/core.go

```diff
@@ -100,8 +106,16 @@ func NewCore(db ethdb.Database, config *Config, isLocalBlock func(block *types.W
 	proccesingCache, _ := lru.NewWithExpire(c_processingCache, time.Second*60)
 	c.processingCache = proccesingCache
 
+	remoteTxQueue, _ := lru.New(c_maxRemoteTxQueue)
+	c.remoteTxQueue = remoteTxQueue
+
+	localTxQueue, _ := lru.New(c_maxLocalTxQueue)
+	c.localTxQueue = localTxQueue
+
 	go c.updateAppendQueue()
 	go c.startStatsTimer()
+	go c.startRemoteTxQueue()
```

Comment:
this should only run in the case of Zone context, i noticed a bunch of panics in Prime and Region
same applies for the localtxqueue